app/lib/account/backend.dart

```diff
@@ -111,6 +112,59 @@ Future<User> requireAuthenticatedUser({AuthSource? source}) async {
   return user;
 }
 
+/// Holds the authenticated agent information.
+class AuthenticatedAgent {
+  final String agentId;
+  final String label;
+  final JsonWebToken jwt;
```

Comment:
```suggestion
  /// OIDC `id_token` the request was authenticated with.
  ///
  /// The [agentId] of an [AuthenticatedAgent] have always been authenticated using the [id_token].
  /// Hence, claims on the [id_token] may be used to determine authorization of a request.
  final JsonWebToken id_token;
```

Reply:
Might even worthwhile documenting that:
 * The signature of the `id_token` have been verified.
 * The `audience` and `expiration` of the `id_token` has been verified.